src/app/models.py

```diff
@@ -11,18 +11,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
 from __future__ import unicode_literals
-
 from django.db import models
 from datetime import datetime
 from django import forms
 from django.contrib.auth.models import User
 
-
 class UserID(models.Model):
-    user = models.OneToOneField(User)
-    organisation = models.CharField("Organisation",max_length=64, null=False, blank=False)
-    lastlogin = models.DateField("Last Login",default=datetime.now,blank=True)
-    def __str__(self):
-        return self.user.username
+	user = models.OneToOneField(User)
+	organisation = models.CharField("Organisation",max_length=64, null=False, blank=False)
+	lastlogin = models.DateField("Last Login",default=datetime.now,blank=True)
+	def __str__(self):
+		return self.user.username
```

Comment:
Please use the default 4 space indentation instead of tab. No need to change already written code.